src/components/NavLinks.tsx

```diff
@@ -33,14 +33,14 @@ const NavLinks = () => {
             >
                 {t('app')}
             </NavBarLink>
-            {/* <NavBarLink
+            <NavBarLink
                 id="earn-link"
                 to="/earn"
                 onClick={(e) => handleLinkClick(e, false)}
                 className={location.pathname.startsWith('/earn') ? 'activeLink' : ''}
             >
                 {t('earn')}
-            </NavBarLink> */}
+            </NavBarLink>
```

Comment:
Lets keep this hidden for now, since we may merge this today, and implement the sdk on Monday